lib/faucet.html

```diff
@@ -0,0 +1,182 @@
+<!doctype html>
```

Comment:
Does this belong to a separate repository?

Reply:
I agree, I think we should have a faucet-ui repo (or maybe a derivation on cardano-sl, since that's where the main faucet code is) or something instead of embedding html/vuejs code in iohk-ops.

Reply:
Will move it to `cardano-sl/faucet/frontend` and add a derivation there.